app/models/fb_text.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# Foodbank, associated to many pantries
+class FbText < ApplicationRecord
+  self.table_name = 'fb_text'
```

Comment:
I think we are not supposed to change table name(because fb_text is the table name in DB.), but yeah i am changing model name and renaming file name.

Reply:
Ideally yes we would have pluralized and consistent table names.  This started with the table foodbanks_mini the model for which was renamed foodbank.    And then the table CNTY was renamed foodbank_county.  I thought it was better to keep the model naming consistent and descriptive as it is used throughout the application.   I think David offered to rename the table but we would still have other inconsistencies, e.g., locations is agency, etc.